src/controllers/order.controllers.js

```diff
@@ -0,0 +1,24 @@
+const postOrderService = require('../services/order.service');
+
+const postOrder = async (req,res) => {
+    try{
+        
+        const { user_id, order_number, status_id,  price, quantity } = req.body;
+
+        if( !order_number || !status_id || !price || !quantity ) {
+            throw new Error("Key Error")
+        }
+        
+        await postOrderService.postOrder(user_id, order_number ,status_id, price, quantity);
+
+        return res.status(200).json({message:"Add Order Success!!"})
```

Comment:
```suggestion
        return res.status(201).json({message:"Add Order Success!!"})
```
order가 만들어졌다면 200이 아닌 201이 적합할 것 같습니다